### DISCUSSION TITLE
Recent branch of ComfyUI is Obsessed with Character Head Portraits

### DISCUSSION THREAD

**Discussion by lord-lethris** [Comfy-Org/ComfyUI#1138] [Q&A]
No matter what I put in the prompt and no matter what Checkpoint models I select AND no matter what weights I apply to the prompts - ComfyUI after a recent update (git pull) seems to do nothing but Female Head shots/Portraits.  I can even type "A teadybear on the moon" - and ComfyUI will do me a portrait of a girl...  its really REALLY odd.

don't get me wrong, the quality is fantastic... its just not what I want 😁

Anyone else seeing this?

it "looks" like its some attempt at better eyes (maybe), but you can see as it renders the image, it keeps flicking between what I actually want and some "Head" photo...

Here you can see it start to grow...
![image](https://github.com/comfyanonymous/ComfyUI/assets/1736720/d0f2b0dc-8614-412f-aca4-b7b2a3bc8e87)

...and there she is!
![image](https://github.com/comfyanonymous/ComfyUI/assets/1736720/1a99c82e-a99b-45ff-9736-0fe736e252f6)

**Comment by Krygio** [comment]
It's hard to imagine that the UI would influence the result that much.
I tested the teddybear, it worked well.
Maybe your prompt is somehow buffered in the workflow?
![ComfyUI_00804_](https://github.com/comfyanonymous/ComfyUI/assets/141618232/a5145e58-f4ab-47e2-98ab-57a47de9cfec)

**Reply by lord-lethris** [reply]
Yea, it's super odd.

I've triple checked my workflow.  I might have to place some debugging stuff around the flow to try to see what happening.

Just wondered if anyone came up with a similar issue.

**Reply by Ferniclestix** [reply]
could be some over training in a lora or hypernetwork, I assume you have used it before though without this issue.

**Comment by ntdviet** [comment]
It's your workflow. Somewhere down the line, your CLIP got wrecked, so the checkpoints (many of them are hentais) revert back to their default if there is an empty CLIP: girls
Personally, I don't like the workflow you use. Once you need to debug, the layout needs to be ripped apart. It looks pretty as a static dashboard, but hard to improve/update/debug

**Reply by lord-lethris** [reply]
Actually - it was my styles setup...  this one to be precise:

`sharp focus, 4k, realistic, masterpiece, best quality, ultra high res, (photorealistic:1.4), Intricate, High Detail, remarkable color, ultra realistic`

For some weird reason - it just draws faces...  maybe its the "photorealistic" weight?

**Reply by Ferniclestix** [reply]
masterpiece, it makes paintings,  try these prompts out by themselves to see what they actually do.

**Reply by lord-lethris** [reply]
that's good to know (re: masterpiece)

Is there a spreadsheet somewhere of common prompt words, like "masterpiece", and AI's interpretation of them?

**Reply by Ferniclestix** [reply]
just assume most prompt words off reddit and civitai are wrong.
remember, models are trained off 4 billion images inspected and labelled by image recognition software, it also uses image metadata to an extent.
you can in many cases google a word and literally see the kind of images it might produce. for instance naming cameras makes it want to put cameras in your image, using UHD makes it want to put labels in the corner saying UHD and so on.

**Comment by ntdviet** [comment]
> Actually - it was my styles setup... this one to be precise:
> 
> `sharp focus, 4k, realistic, masterpiece, best quality, ultra high res, (photorealistic:1.4), Intricate, High Detail, remarkable color, ultra realistic`
> 
> For some weird reason - it just draws faces... maybe its the "photorealistic" weight?

No, "photorealistic:1.4" is just fine.
If the problem is eliminated by cutting off the styles, maybe it's your style loader buggy, or wrongly wired. You may post your workflow, it's easier to see your issue or replicate it.
